addons/mass_mailing/models/mailing.py

```diff
@@ -829,6 +830,7 @@ def action_select_as_winner(self):
         })
         final_mailing = self.copy({
             'ab_testing_pc': 100,
+            'ab_testing_is_winner_mailing': True,
```

Comment:
maybe to be discussed with LNA, but shouldn't we merge all the domain of the mailings if they are different with an `OR` ?

Reply:
Leaving as is as per lna's point on the task (different models -> different wording in mailings)